docker/Dockerfile

```diff
@@ -0,0 +1,166 @@
+ARG ROS_VERSION=jazzy
+ARG UBUNTU_VERSION=noble
+
+FROM ros:${ROS_VERSION}-ros-base-${UBUNTU_VERSION}
```

Comment:
You can use `ros:humble-ros-base` or `ros:jazzy-ros-base` tags and eliminate the Ubuntu version argument.